tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -366,10 +391,33 @@ def test_attention_mask(
     assert np.all(mask_zeros == 0)
 
 
-# TODO: need to fix this failing test
-@pytest.mark.skip(reason="Results in random crashing of github action workers")
+def skip_on_CI(model_name, model_weights):
+    # This only applies when skip_model_load=False
+    return model_name == "bert" and bool(os.environ.get("CI")) and not model_weights
+
+
+def create_config(model_name, model_weights):
+    """
+    Create a config for LanguageModelFeaturizer. Skips model/model_weight
+    combinations that are too large (bert with LaBSE).
+
+    Args:
+        model_name: model name
+        model_weights: model weights name
+    """
+    if skip_on_CI(model_name, model_weights):
+        pytest.skip(
+            "Reason: this model is too large, loading it results in"
+            "crashing of GH action workers."
+        )
```

Comment:
Will calling this in a separate function also skip the pytest?

Reply:
Yep! I was sceptical of this too, but you can see in the logs four tests are skipped (three for `LanguageModelFeaturizer` and one for `HFTransformersNLP`). You can also test this locally for yourself if you just add the line `os.environ["CI"] = "true"`